store-implementation/parquet-store/src/test/java/uk/gov/gchq/gaffer/parquetstore/utils/ConvertViewToFilterTest.java

```diff
@@ -75,7 +83,8 @@ public void getMultiColumnGroupFilterTest() throws OperationException, IOExcepti
                                 .build())
                         .build())
                 .build();
-        final FilterPredicate filter = ParquetFilterUtils.buildGroupFilter(view, schemaUtils, "BasicEntity", DirectedType.EITHER, true).get0();
+        filterUtils.buildPathToFilterMap(view, DirectedType.EITHER, SeededGraphFilters.IncludeIncomingOutgoingType.EITHER, SeedMatching.SeedMatchingType.EQUAL, new ArrayList<>(), new GraphIndex());
+        final FilterPredicate filter = filterUtils.buildGroupFilter("BasicEntity", true);
```

Comment:
Rather than hardcoding BasicEntity there is a constant TestGroups.ENTITY that equals "BasicEntity"